features/admin.users.v1/components/edit-user.tsx

```diff
@@ -107,6 +108,10 @@ export const EditUser: FunctionComponent<EditUserPropsInterface> = (
     const [ isUserManagedByParentOrg, setIsUserManagedByParentOrg ] = useState<boolean>(false);
     const [ isUserProfileReadOnly, setIsUserProfileReadOnly ] = useState<boolean>(false);
 
+    const userRolesDisabledFeatures: string[] = useSelector((state: AppState) => {
+        return state.config.ui.features?.users?.disabledFeatures;
+    });
```

Comment:
Since this should only affect the admin users, the disabled feature identifier should be added under console settings, shouldn't it?

Reply:
This is a general configuration to hide the Groups tab in the user profile. Therefore, we have added the configuration under 'users'.

We followed a similar approach as implemented in [#6373](https://github.com/wso2/identity-apps/pull/6373) to maintain consistency.